.github/workflows/ci.yml

```diff
@@ -79,6 +79,48 @@ jobs:
           python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
           chmod a+x builder
           ./builder build -p ${{ env.PACKAGE_NAME }} --target=linux-${{ matrix.arch }} --spec=downstream
+          
+  linux-musl:
+    runs-on: ubuntu-22.04 # latest
+    strategy:
+      matrix:
+        image:
+          - alpine-3.16-x64
+          - alpine-3.16-x86
```

Comment:
debatable: should we bother testing this? we don't package it up
```suggestion
```

Reply:
Maybe it is worth keeping it in CI so that we know that we are not breaking anything for 32-bit Musl. We don't specifically package OpenWrt or Raspberry either.

Reply:
those were introduced because we knew we've had customers using them, and fixed bugs, and added those CI passes as part of the bugfix. I'm not aware of anyone using 32bit musl. I'd rather not add complexity with no known beneficiaries. There's an infinite number of platforms to support if that's the case

Reply:
Thanks, makes sense. We can add it when we know there are customers using it.